src/System.ServiceModel.NetTcp/ref/project.json

```diff
@@ -3,7 +3,7 @@
     "System.ObjectModel": "4.0.0",
     "System.Runtime": "4.0.0",
     "System.Net.Primitives": "4.0.10",
-    "System.ServiceModel.Primitives": "4.1.0-beta-*",
+    "System.ServiceModel.Primitives": "4.1.0-rc2-23602",
```

Comment:
Does this build in TFS?  I'm having TFS build issues right now and can't verify.